fdbclient/BlobGranuleReader.actor.cpp

```diff
@@ -256,7 +256,9 @@ static void applyDeltas(std::map<KeyRef, ValueRef>* dataMap,
 ACTOR Future<RangeResult> readBlobGranule(BlobGranuleChunkRef chunk,
                                           KeyRangeRef keyRange,
                                           Version readVersion,
-                                          Reference<BackupContainerFileSystem> bstore) {
+                                          Reference<BackupContainerFileSystem> bstore,
+                                          Optional<BlobWorkerStats *> stats) {
+
```

Comment:
Could also default this to nullptr and not use `Optional` but `Optional` seems like a cleaner construct.

Reply:
Using `Reference` will require inheriting from `ReferenceCounted` so we will leave this for now